tests/Base.php

```diff
@@ -34,6 +34,16 @@ protected function extractName($value) {
         return $value['name'];
     }
 
+    protected function extractRoomFromArray($roomID, $rooms) {
```

Comment:
I feel like there must be a more elegant way to do this, I'll check the PHP docs

Reply:
doesn't seem like it :/

Reply:
:joy: